sample/wallet/src/main/kotlin/com/walletconnect/sample/wallet/ui/routes/dialog_routes/session_proposal/SessionProposalRoute.kt

```diff
@@ -214,6 +211,23 @@ fun AccountAndNetwork(sessionProposalUI: SessionProposalUI) {
     }
 }
 
+private fun getOptionalChains(sessionProposalUI: SessionProposalUI) =
+    sessionProposalUI.optionalNamespaces.flatMap { (namespaceKey, proposal) ->
```

Comment:
not stylish enough. Please use the same style as in `getRequiredChains`
```suggestion
private fun getOptionalChains(sessionProposalUI: SessionProposalUI) = sessionProposalUI.optionalNamespaces.flatMap { (namespaceKey, proposal) ->
```